torch_em/model/unetr.py

```diff
@@ -142,8 +144,51 @@ def __init__(
                 window_size=14,
                 out_chans=256,
             )
+
+        elif encoder == "vit_l":
+            self.encoder = ViT_Sam(
+                depth=24,
+                embed_dim=1024,
+                img_size=1024,
+                mlp_ratio=4,
+                norm_layer=partial(torch.nn.LayerNorm, eps=1e-6),  # type: ignore
+                num_heads=16,
+                patch_size=16,
+                qkv_bias=True,
+                use_rel_pos=True,
+                global_attn_indexes=[5, 11, 17, 23],  # type: ignore
+                window_size=14,
+                out_chans=256
+            )
+
+        elif encoder == "vit_h":
+            self.encoder = ViT_Sam(
+                depth=32,
+                embed_dim=1280,
+                img_size=1024,
+                mlp_ratio=4,
+                norm_layer=partial(torch.nn.LayerNorm, eps=1e-6),  # type: ignore
+                num_heads=16,
+                patch_size=16,
+                qkv_bias=True,
+                use_rel_pos=True,
+                global_attn_indexes=[7, 15, 23, 31],  # type: ignore
+                window_size=14,
+                out_chans=256
+            )
+
         else:
-            self.encoder = encoder
+            raise NotImplementedError
+
+        if initialize_from_sam:
```

Comment:
I think we don't need `initialize_from_sam`. Isn't it enough to have `if checkpoint_path is not None` here? (And then you don't need to have the assertion.)